intranethome/templates/intranethome/mail_aliases.html

```diff
@@ -0,0 +1,178 @@
+<style>
+      .accordion {
+        background-color: #eee;
+        color: #444;
+        cursor: pointer;
+        padding: 18px;
+        width: 100%;
+        border: none;
+        text-align: left;
+        outline: none;
+        font-size: 15px;
+        }
+    
+      .active, .accordion:hover {
+        background-color: #ccc;
+        }
+    
+      .accordion:after {
+        content: '\002B';
+        color: #777;
+        font-weight: bold;
+        float: right;
+        margin-left: 5px;
+        }
+    
+      .active:after {
+        content: "\2212";
+        }
+    
+      .panel {
+        padding: 0 18px;
+        background-color: white;
+        overflow: hidden;
+        }
+</style>
+
+<body>
+
+{% extends "base/intranet_base.html" %}
+
+{% block body_class %}template-searchresults{% endblock %}
+
+{% block title %}Mail Aliases{% endblock %}
+
+{% block content %}
+
+<h1>Mail Aliases Page</h1>
+
+{%if error%}
+	There has been an error displaying the mailaliases page. Please contact <a href "mailto:intranet@lib.uchicago.edu">intranet@lib.uchicago.edu</a> and we will do our best to rectify the issue.
+{%else%}
```

Comment:
This template should be linted with:

`curlylint --parse-only intranethome/templates/intranethome/mail_aliases.html`

and 

`djhtml -i intranethome/templates/intranethome/mail_aliases.htm`

curlylint is reporting a syntax error. I haven't determined if it's a real syntax error or just a false positive. It might just be that you need spaces after the `%`s. I'm not sure.